src/utils/formatLocalisedCompactNumber.ts

```diff
@@ -0,0 +1,12 @@
+import { getLanguageCodeFromLS } from 'contexts/Localization/helpers'
```

Comment:
not a big fan of creating a new file at the root for every helper. we already have several formatting helpers, maybe it's time to create a folder for this

Reply:
Sure - I've pushed it into `formatBalance` which has a lot of general number formatting helpers. It should maybe be renamed 'numberFormatters' or something in a separate PR?